src/main/fc/fc_msp.c

```diff
@@ -1032,6 +1032,64 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, sbuf_t *src, msp
         sbufWriteU8(dst, masterConfig.gyroSync);
         break;
 
+    case MSP_FILTER_CONFIG :
```

Comment:
hmm.... tough call... I think I will add it to MSP_INAV_PID frame. There are still reserved bytes